### PR TITLE
update to 5.0.3

### PR DESCRIPTION
**Pull request by katietz**
* update to 5.0.3

sadly, version 5.0.2 is no longer avaiable in internet.  so for python 3.11 build-out we need to update it to newer version (with some bug-fixes)

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: ffea98b29c6a7d25a4296eed400b491c89fd89365806a0f646ede44c043727fa
+  sha256: 2586bcb61d29d76e89e563a6836df24d86939961c9657f129a59f7617de45377
 
 build:
-  noarch: python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:13]: Skip `py<36`, https://github.com/hsluv/hsluv-python/blob/b628f927d89969c5f046cceededc68b28617076c/setup.py#L29

**Review by skupr-anaconda — COMMENTED** [review]
Adding `pip check` would be nice

**Review by cbouss — COMMENTED** [review]
Good to merge, with the changes proposed by Serhii

File: recipe/meta.yaml (hunk 1 above)
Comment (katietz) [reply recipe/meta.yaml:13]: ok, thanks

**Pull request merged**